dataland-frontend/tests/e2e/utils/LksgUpload.ts

```diff
@@ -219,7 +219,7 @@ function fillNewProduct(name: string, productionSteps: string[], description?: s
 /**
  * Adds some products to the most important products form
  */
-function fillInMostImportantProducts(): void {
+export function fillInMostImportantProducts(): void {
```

Comment:
Why are you exporting this. It is only used in this file here (at least I cannot find any other occurrence)

Reply:
Done.

Reply:
Changed it